addons/configs/config.cpp

```diff
@@ -50,10 +50,20 @@ class CfgPatches {
         hiddenSelectionsTextures[] = { paa }; \
     }
 
-#define HEADGEAR(name, base, text, paa) \
+#define HEADGEAR(name, base, text, paas) \
```

Comment:
"paas" , should this be "paa"?

Reply:
I just uploaded the files we've been using since mid-2019. Dunno why Price changed this but it hasn't impacted the mod in the past several months so I assume it's changed for a reason.

Reply:
fair enough